exercises/giovanni_disiena/ex02_oo_basics/src/Use.h

```diff
@@ -0,0 +1,16 @@
+#ifndef USE_H
+#define USE_H
+
+class Use {
+public:
+  Use();
+  Use(const Use &u);
+  ~Use();
+  Use &operator=(const Use &);
+  operator size_t() const;
+
+private:
+  unsigned *m_refCount;
+};
+
+#endif // USE_H
```

Comment:
This is also not in the `CMakeLists` and not used anywhere, you need to remove the file if you are not including it.